src/metrics/generated/proto/pipelinepb/pipeline.proto

```diff
@@ -87,5 +87,7 @@ message AppliedPipelineOp {
 // AppliedPipelineOp is a pipeline containing operations
 // that have been applied against a metric.
 message AppliedPipeline {
+  // A custom Unmarshal function exists in custom_unmarshal.go to allow reusing slices.
+  option (gogoproto.unmarshaler) = false;
```

Comment:
Is this meant for another commit? from the commit message, it looks like you were trying to break this up.

Reply:
no. this point of this commit is to stop auto-generating an Unmarshal method and copy the existing generated Unmarshal to a new file. you can't have both co-exist or go will complain.

Reply:
Cool. Thanks.